dailySales.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="icon" href="img/logo.png" type="image/x-icon">
+    <link rel="stylesheet" href="css/custom.css" type="text/css">
+    <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
+
+    
+    <title>Store House</title>
+  </head>
+  <body>
+
+    <div class="container">
+            <form action="" class="my-5 mx-5">
+                <input type="text" name="item" id="drug-name" placeholder="Drug name">
+                <input type="text" name="price" id="drug-price" placeholder="cost">
+                <input type="button" name="add-item" class="add-item" value="Add Item" >
+
+            </form>
+
+    <table>
+        <thead>
+        <tr class="lead container bg-primary">
+            <th>Select</th>
+            <th>S/No.</th>
+            <th>Drug name</th>
+            <th>Price</th>
+            <th>Payment Status</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="text-muted">
```

Comment:
Please remove the text-muted class. There's no need to make the first table row text-muted.

Reply:
correction has been made excluding the  addition of the total